Flutter/funnel-chart/export-funnel-chart.md

```diff
@@ -18,57 +18,246 @@ To export the funnel chart as a PNG image, we can get the image by calling [`toI
+      Widget build(BuildContext context) {
+        return const MaterialApp(
+            home: ExportFunnel(),
+        );
+      }
+    }
+    
+    class ExportFunnel extends StatefulWidget {
+      
+      const ExportFunnel({Key? key}) : super(key: key);
+
+      @override
+      _ExportState createState() => _ExportState();
+    }
+
+    class _ExportState extends State<ExportFunnel> {
+      _ExportState();
+
+      late GlobalKey<SfFunnelChartState> _funnelChartKey;
+      late List<ChartSampleData> _chartData;
+
+      @override
+      void initState() {
+        _funnelChartKey = GlobalKey();
+        _chartData = <ChartSampleData>[
+          ChartSampleData(x: 'Jan', y: 12),
+          ChartSampleData(x: 'Feb', y: 28),
+          ChartSampleData(x: 'Mar', y: 35),
+          ChartSampleData(x: 'Apr', y: 47),
+          ChartSampleData(x: 'May', y: 56),
+          ChartSampleData(x: 'Jun', y: 70),
+        ];
+        super.initState();
+      }
+
+      @override
+      Widget build(BuildContext context) {
+        return Scaffold(
+          body: Column(
+            mainAxisAlignment: MainAxisAlignment.spaceEvenly,
+            children: <Widget>[
+              SfFunnelChart(
+                key: _funnelChartKey,
+                series: FunnelSeries<ChartSampleData, String>(
+                    dataSource: _chartData,
+                    xValueMapper: (ChartSampleData data, _) => data.x,
+                    yValueMapper: (ChartSampleData data, _) => data.y,
+                  )
+              ),
+              TextButton(
+                child: const Text('Export as image'),
+                onPressed: () {
+                  _renderFunnelImage();
+                },
+              )
+            ]
+          ),
+        );
+      }
+
+      Future<void> _renderFunnelImage() async {
+        final ui.Image data =
+          await _funnelChartKey.currentState!.toImage(pixelRatio: 3.0);
+        final ByteData? bytes =
+          await data.toByteData(format: ui.ImageByteFormat.png);
+        final Uint8List imageBytes =
+          bytes!.buffer.asUint8List(bytes.offsetInBytes, bytes.lengthInBytes);
+        if (!mounted) return;
+        await Navigator.of(context).push<dynamic>(
+          MaterialPageRoute<dynamic>(
             builder: (BuildContext context) {
               return Scaffold(
                 appBar: AppBar(),
-                body: Center(
-                  child: Container(
-                    color: Colors.white,
-                    child: Image.memory(bytes!.buffer.asUint8List()),
-                  ),
-                ),
-              );
+                body: Image.memory(imageBytes));
             },
           ),
         );
       }
     }
 
-  {% endhighlight %}
+    class ChartSampleData {
+      ChartSampleData({this.x, this.y});
+        final String? x;
+        final num? y;
+    }
+
+{% endhighlight %}
 {% endtabs %}
 
+![image_export](images/export-funnel-chart/image_view.gif)
```

Comment:
<img width="339" alt="image" src="https://user-images.githubusercontent.com/92971875/204241481-32de24d4-0bef-4975-b5df-858792633f88.png">

Form a git like this